trunk/NDHMS/Data_Rec/module_namelist.F

```diff
@@ -53,6 +53,11 @@ subroutine read_rt_nlst(nlst)
           character(len=256) :: route_lake_f=""
           character(len=256) :: route_direction_f=""
           character(len=256) :: route_order_f=""
+          logical            :: persistence
```

Comment:
I would suggest changing this name to reservoir_persistence or res_persistence. 
The nudging persistence is in its own namelist, but this is not. I think the variable name needs to communicate what is being persisted.

Reply:
Likewise for the next line for "persistence_parameter_file". The subsequent variables start with "reservoir_" which is 👍

Reply:
Okay will change this too.